src/govuk/components/character-count/character-count.mjs

```diff
@@ -229,13 +273,13 @@ CharacterCount.prototype.count = function (text) {
  */
 CharacterCount.prototype.getCountMessage = function () {
   var $textarea = this.$textarea
-  var options = this.options
+  var config = this.config
   var remainingNumber = this.maxLength - this.count($textarea.value)
 
   var charVerb = 'remaining'
   var charNoun = 'character'
   var displayNumber = remainingNumber
-  if (options.maxwords) {
+  if (config.maxwords) {
```

Comment:
As discussed on Slack, as the local `config` variable is only used once we can probably avoid it:

```suggestion
  if (this.config.maxwords) {
```

(Could do the same for `this.textarea`?)

Reply:
This gets completely revamped in the I18n PR so I'll leave it as is in this one: https://github.com/alphagov/govuk-frontend/pull/2887/files#diff-604c43935cbbf4edb8c1209bc9b06ae7341fdaf64e208f777db723fd456bf9baR294